### DISCUSSION TITLE
info Lampadine come router zigbee

### DISCUSSION THREAD

**Discussion by Borghese** [indomus/forum#2831] [GENERALE - Discussioni generiche]
ciao ragazzi, ho un dubbio che non riesco a dipanare pur leggendo mille guide online... vi spiego : 

Sto iniziando a installare un pò di sensori in casa, linea aqara, zigbee, per rilevare temperatura, umidità, chiusura/aperture porte finestre etc etc...
Il coordinator è la chiaveta usb sonoff con antenna installata direttamente sul mini pc intel nuc che ospita HA, e che sta nello studio.
La casa non è immensa, ma sono comunque due piani, ca 60mq/piano, e lo studio è al primo piano.

Chiaro è quindi che, per es per i sensori al piano terra, faccio fatica a coprire tutto con la rete zigbee, e ho quindi necessità di installare qualcosa che mi faccia da router in mezzo
Ho quindi pensato alle lampadine zigbee (ne ho già preso una per fare dei test)
Ma il dubbio che ho è questo : 

Io ho già tutti i punti  luce domotizzati con switch sonoff, ma a prescindere da questo, le lampadine, per fare da router, devono cmq essere sempre alimentate giusto ? 
Questo significa che, gli interruttori/deviatori/invertitori che comandano queste lampade, devono stare sempre accesi, perchè poi l'accensione/spegnimento della luce si comanda direttamente dalla lampadina ? 
Di contro, se qualcuno spegne l'interruttore, la lampadina non è piu alimentata, non fa piu da router, e men che meno si puo comandare via assistente vocale/app home assistant/automazione, se qualcuno non si alza e accende l'interrutore.

Sto interpretando bene o mi sta sfuggendo qualcosa ?

**Comment by Scialla** [comment]
stai interpretando bene...
puoi anche usare delle prese domotiche, degli alimentatori USB, delle strisce led o, in generale, qualsiasi device zigbee che sia alimentato (che farà da router), es. ciabatte, purificatori, sensori presenza etc

l'uso delle lampadine, per i motivi che hai citato, non è consigliabile, a meno che non rimangono sempre alimentate

**Comment by Borghese** [comment]
grazie Scialla,
si me ne sto rendendo conto, per quanto mi piaccia la funzionalità di dimmer luminosità e colore, che con gli switch sonoff non hai, però questa cosa che poi gli interruttori debbano sempre stare su on in effetti non mi fa impazzire.
Ora ho ordinato una decina di sensori temperatura aqara da aliexpress, li piazzo un pò in tutte le stanze di casa, poi in base ai buchi di copertura della rete zigbee valuterò su quali altri device andare che facciano da router....

grazie intanto per le dritte!
